docs/pangeo_forge_recipes/recipe_user_guide/recipes.md

```diff
@@ -59,29 +60,43 @@ tutorials for this recipe class. These are, in order of increasing complexity
 Below we give a very basic overview of how this recipe is used.
 
 First you must define a {doc}`file pattern <file_patterns>`.
-Once you have a {class}`file_pattern <pangeo_forge_recipes.patterns.FilePattern>` object,
-initializing an `XarrayZarrRecipe` can be as simple as this.
-
+Once you have a {class}`FilePattern <pangeo_forge_recipes.patterns.FilePattern>` object,
+the recipe pipeline will contain at a minimum the following transforms applied to the file pattern collection:
+* `OpenURLWithFSSpec`: retrieves each pattern file using the specified URLs.
```

Comment:
Can we make this a reference to the actual class in the API docs?

Reply:
Done